app/src/main/java/cm/aptoide/pt/app/AppViewAnalytics.java

```diff
@@ -220,6 +221,21 @@ public void sendAppcInfoInteractEvent() {
         AnalyticsManager.Action.CLICK, getViewName(true));
   }
 
+  public void sendDonateClickAfterInstall() {
+    analyticsManager.logEvent(createMapData(ACTION, "Donations after install view"),
+        APP_VIEW_INTERACT, AnalyticsManager.Action.CLICK, getViewName(true));
+  }
+
+  public void sendDonateClickTopDonors() {
+    analyticsManager.logEvent(createMapData(ACTION, "Top donors view"), APP_VIEW_INTERACT,
```

Comment:
According to the ticket the action message should be donate_click_top

Reply:
fixed